procedures/katello/branch.md.erb

```diff
@@ -3,25 +3,41 @@
 ## Release Owner
 
 - [ ] Announce upcoming branching to discourse development category a month before
-- [ ] Start extracting i18n strings and pushing the changes into master git branches so Transifex can pick it up
 - [ ] Update Katello Transifex translations:
   - [ ] Create a Transifex account and join the Foreman team
   - [ ] Spin up a Foreman and Katello installation
-  - [ ] Install Transifex client `pip3 install transifex-client --user`
-  - [ ] [Configure the Transifex client](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
+  - [ ] Install Transifex client: `pip3 install transifex-client --user` (make sure to pass `--user` flag!)
+  - [ ] Configure `~/.transifexrc` [as outlined here](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
   - [ ] [Install grunt](https://github.com/Katello/katello/blob/master/engines/bastion_katello/README.md)
   - [ ] `grunt i18n:extract` in the katello/engines/bastion_katello directory
   - [ ] `make -C locale tx-update` in the katello directory
   - [ ] `grunt i18n:compile` in the katello/engines/bastion_katello directory
   - [ ] `bundle exec rake plugin:gettext[katello]` in the foreman directory
   - [ ] Commit the resulting files in the katello directory with a message like "i18n - pulling from tx"
+  - [ ] Open a PR to Katello (no Redmine issue needed)
+- [ ] Start attending upstream release sync meetings and giving updates
+
+# Three Weeks Prior to Branch Date
+
+## Release Owner
+
+- [ ] Ensure that issues requiring installer changes are merged
 
 # Two Weeks Prior to Branch Date
 
 ## Release Owner
 
-- [ ] Add tool_belt config for new release [tool_belt configs](https://github.com/theforeman/tool_belt/tree/master/configs/katello)
-  - [ ] Check the nightly config file if any repos/tags need to be updated in the newly created config.
+- [ ] Add tool_belt config for new release:
+  - [ ] Create a new yaml file using the previous Katello version as a template: [tool_belt configs](https://github.com/theforeman/tool_belt/tree/master/configs/katello)
```

Comment:
Using the previous is always a bad idea IMHO. `nightly` should actually be more future proof I think. In the past we've seen that modifications made for nightly were forgotten because it copied it from the existing release.